Sources/Classes/RSClient.m

```diff
@@ -506,12 +506,12 @@ - (NSNumber * _Nullable)sessionId {
     return (_repository != nil) ? [_repository getSessionId] : nil;
 }
 
-- (void) onIntegrationReady:(NSString*)key withCallback:(Callback)callback {
+- (void) onIntegrationReady:(NSString*)integrationName withCallback:(Callback)callback {
```

Comment:
Shall we also add one more method where the user can pass the factory instance, and we internally retrieve the integration name from the factory and use it? (This will make the job easier for the customers, otherwise, they need to make sure they are passing the exact destination name that we are defining in the device mode factory)

Reply:
They can simply do this: `[RudderBrazeFactory instance].key` to get the appropriate key. Will that not be enough?
If you still suggest implementing the method please let me know, and I'll implement it.

Reply:
Yes' let's retrieve the key ourselves when the customers passes the instance of the factory.